api/v1/datanetwork_types.go

```diff
@@ -109,7 +109,7 @@ type DataNetworkStatus struct {
 // attributes of a StarlingX system.  This is a composition of the following
 // StarlingX API endpoints.
 //
-//   https://docs.starlingx.io/api-ref/stx-config/api-ref-sysinv-v1-config.html#data-networks
+//	https://docs.starlingx.io/api-ref/stx-config/api-ref-sysinv-v1-config.html#data-networks
```

Comment:
I noticed this kind of format change during reviewing the package update in Ritik's recent commit. Is it introduced automatically?

Reply:
I'm using MS code and it automatically formatted all codes like this every time I open and save (even the code which I did not modify). I could put it back the format, so please let me know, thank you

Reply:
Is it changed recently? If that's a one-time deal and will not be changed the next time, I'll prefer to leave it changed as multiple people hit it. Changing a lot of formats is difficult to review.